LBH.AdultSocialCare.Functions.Payruns/Services/InvoiceGenerator.cs

```diff
@@ -103,7 +114,7 @@ private async Task InitializeGeneratorsAsync()
                     }
                 },
                 {
-                    PackageType.NursingCare, new List<BaseInvoiceItemsGenerator>
+                    PackageType.ResidentialCare, new List<BaseInvoiceItemsGenerator>
```

Comment:
Why `FundenNursingCareGenerator` is not part of ResidentialCare?

Reply:
There is no restriction on package-brokerage. Also, I've confirmed with Martin, it is possible :'(

Reply:
confusion on this topic is clarified, merging